history-of-the-quakers/original/28-book-3-chapter-2.adoc

```diff
@@ -220,7 +222,9 @@ A copy of a letter from Leicester jail,
 dated the 20th of the twelfth month,
```

Comment:
There are two letters in this footnote. I didn't know if I could use the normal formatting, and if not, should I put them in quotes at all or anything?

Reply:
haha, i've never seen a full letter in a footnote. no, you can't embed, I think you did the right thing. this doc is crazy!